code/modules/ninja/energy_katana.dm

```diff
@@ -116,5 +116,5 @@
 /datum/action/innate/dash/ninja
 	current_charges = 3
 	max_charges = 3
-	charge_rate = 200
+	charge_rate = 350 //SKYRAT EDIT CHANGE - ORIGINAL: 200
```

Comment:
so is there a reason its 350 and not like... 35 SECONDS?

Reply:
No, but TG

Reply:
ah